docs/messaging-implementation.md

```diff
@@ -66,6 +71,8 @@ Widget messaging is based on JSON input configured in a [widget's configuration]
     You would use this feature if you want to only show the message if the specific user has data. For example: Only show user if they have a certain document.
   - **dataObject**: *(optional)* Will only be looked at if `dataUrl` is present, otherwise ignored. Used as an optional further refinement from dataUrl, if you want the notification to show only if the specific object is in the data.
   - **dataArrayFIlter**: Will only be looked at if `dataUrl` is present, otherwise ignored. Used as an optional further refinement from dataUrl. If your object return is an array, you can filter on the array. Does support multiple filtering criteria as shown in the example. If used in conjunction with `dataObject`, will filter to `dataObject` first.  [AngularJS array filtering documentation] (https://docs.angularjs.org/api/ng/filter/filter)
+  - **dataMessageTitle** Will be used if dataUrl is specified. Used to set the title of the message from the data response from `dataUrl`.  Expects an array for where to find the title in the data response from `dataUrl`.
```

Comment:
:question: would it be easier if there was a `data` block in the json with `Object`, `ArrayFilter`, `MessageTitle`, and `MessageMoreInfoUrl` as child properties?

Reply:
Good insight.  Yeah, probably.

Reply:
Moved to a new `data` block.